src/ExternalResource.ts

```diff
@@ -90,11 +89,40 @@ namespace Manifold {
 
         private _parseAuthServices(resource: any): void {
 
+            let auth09_clickThroughService: Manifesto.IService | null = manifesto.Utils.getService(resource, manifesto.ServiceProfile.clickThrough().toString());
+            let auth09_loginService: Manifesto.IService | null = manifesto.Utils.getService(resource, manifesto.ServiceProfile.login().toString());
+            let auth09_restrictedService: Manifesto.IService | null = manifesto.Utils.getService(resource, manifesto.ServiceProfile.restricted().toString());
+
+            let auth1_clickThroughService: Manifesto.IService | null = manifesto.Utils.getService(resource, manifesto.ServiceProfile.auth1Clickthrough().toString());
+            let auth1_loginService: Manifesto.IService | null = manifesto.Utils.getService(resource, manifesto.ServiceProfile.auth1Login().toString());
+            let auth1_externalService: Manifesto.IService | null = manifesto.Utils.getService(resource, manifesto.ServiceProfile.auth1External().toString());
+            let auth1_kioskService: Manifesto.IService | null = manifesto.Utils.getService(resource, manifesto.ServiceProfile.auth1Kiosk().toString());
+
+            if ( auth09_clickThroughService || auth09_loginService || auth09_restrictedService ) {
+
+              this.authAPIVersion = 0.9;
+
+            }
+            else if (
+
+                auth1_clickThroughService || auth1_loginService || auth1_externalService || auth1_kioskService

```

Comment:
Could this be simplified? Just check for existence of auth1 services and default to auth0.9:

```
if (auth1_clickThroughService || auth1_loginService || auth1_externalService || auth1_kioskService) {
    this.authAPIVersion = 1;
} else {
    this.authAPIVersion = 0.9;
}